conda_forge_tick/migrators.xsh

```diff
@@ -89,6 +95,10 @@ class Migrator:
         """Branch to use on local and remote"""
         return 'bot-pr'
 
+    def migrator_hash(self, attrs):
+        return {'class': self.__class__.__name__,
+                'class_version': self._class_version}
```

Comment:
Making this a dict complicates searching with whoosh. I can probably write a new whoosh field type for list of dict, but we need to decide what key names to use. Or maybe we don't care about searching PRed and it can be left out of the schema.

Reply:
Hmm, how are we going to handle multi-source packages, don't they have a list of dicts?
I wouldn't worry about searching this info yet. This will most likely need to go through a refactor when libcflib has a feedstocks graph.

Reply:
Multi-source packages have a list of dicts for what field?

Reply:
source https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-multiple-sources